specs/babbage/crypto.cddl

```diff
@@ -31,6 +26,43 @@ pool_metadata_hash    = $hash32
 ;   "\x00" for multisig scripts
 ;   "\x01" for Plutus V1 scripts
 ;   "\x02" for Plutus V2 scripts
-scripthash            = $hash28
+;scripthash            = $hash28
+
+;datum_hash = $hash32
+
+KES_signature /= bytes .size 32
+signkeyKES = bytes .size 16
+
+VRF_cert /= [bytes, bytes .size 10]
```

Comment:
these are definitely not the right sizes. I'll update this later using the old CML/CSL as a reference as those should have the right sizes.